src/custom/components/AccountDetails/Transaction/StatusDetails.tsx

```diff
@@ -37,6 +38,51 @@ export function GnosisSafeLink(props: {
   return <ExternalLink href={safeUrl}>View Gnosis Safe ↗</ExternalLink>
 }
 
+function _getStateLabel({
+  isPending,
+  isOrder,
+  isConfirmed,
+  isExpired,
+  isCancelling,
+  isPresignaturePending,
+  isCancelled,
+  enhancedTransaction,
+}: ActivityDerivedState) {
+  if (isPending) {
+    if (enhancedTransaction) {
+      console.log('enhancedTransaction', enhancedTransaction)
+      const { safeTransaction, transactionHash } = enhancedTransaction
+      if (safeTransaction && !transactionHash) {
+        return 'Signing...'
+      }
+    }
+
+    return isOrder ? 'Open' : 'Pending...'
+  }
+
+  if (isConfirmed) {
+    return 'Filled'
+  }
```

Comment:
It has been reported elsewhere by @elena-zh and I believe this is the culprit.

Wrap and Approve txs now show as `Filled`.

Just and `if` away for being fixed ;)